drivers/ethernet/dsa_ksz8xxx.c

```diff
@@ -682,7 +701,7 @@ static int dsa_ksz8xxx_configure_bus(struct ksz8xxx_data *pdev)
 	/* SPI config */
 	pdev->spi = (struct spi_dt_spec) SPI_DT_SPEC_INST_GET(0,
 #if DT_INST_PROP(0, spi_cpol)
-		SPI_MODE_CPOL |
+		SPI_MODE_CPOL | 
```

Comment:
This is probably not needed (not related to this patch series).
You can do the whitespaces "cleanup" as a separate commit.